SharpPcap/CaptureEventArgs.cs

```diff
@@ -26,18 +26,29 @@ namespace SharpPcap
     /// <summary>
     /// Capture event arguments
     /// </summary>
-    public class CaptureEventArgs : EventArgs
+    public ref struct CaptureEventArgs
     {
+        private RawCapture rawCapture;
+
         /// <summary>
         /// Packet that was captured
         /// </summary>
-        public RawCapture Packet { get; }
+        public RawCapture Packet {
```

Comment:
I think it would be best to change this into a `GetPacket()` method, to avoid the side effects of modifying a struct